File: acm_statistic_backend/api/api/DTO/utilisateur/UtilisateurDTO.cs
using api.Model.exploitants;

namespace api.DTO.utilisateur
{
    public class UtilisateurDTO
    {
        public String? UserCode { get; set; }

        
        public String? UserName { get; set; }
        public String? UserTitle { get; set; }
        public String? UserEmail { get; set; }
        public String? Role { get; set; }

        public string? UserPasswordString { get; set; }

        public String? IdExploitant { get; set; }
        public virtual Exploitant? Exploitant { get; set; }
        public String? Telephone { get; set; }

    }
}

File: acm_statistic_frontend/Interface Tableau de Bord Analytique Aérien/src/models/utilisateurs/UtilisateurDTO.ts
import { ExploitantDTO } from "../exploitants/ExploitantDTO";
export class UtilisateurDTO {
  userCode?: string;
  userName?: string;
  userTitle?: string;
  userEmail?: string;
  role?: string;
  userPasswordString?: string;
  idExploitant?: string;
  exploitant?: ExploitantDTO;
  telephone?: string;

  constructor(init?: Partial<UtilisateurDTO>) {
    Object.assign(this, init);
  }
  isValid(): boolean {
    return !!this.userCode && !!this.userPasswordString;
  }
}